### PR TITLE
Working cellpose-python and stardist tests

### PR DESCRIPTION
**Pull request by jmetz**
NOTE: This is not ready to merge

### CODE REVIEW

**Comment by jmetz** [comment]
Hey @oeway - I think this looks mostly reasonable now (though let me know if you want me to remove imjoy.io from the server list).

File: scripts/test-triton-models.py [hunk 1]
```diff
@@ -29,4 +45,37 @@ async def main():
...
+    # One-off test of the cellpose model
+    image = imageio.v3.imread('https://static.imjoy.io/img/img02.png')
+    try:
+        ret = await triton.execute(
+            inputs=[image.astype('float32'), {'diameter': 30}],
+            model_name="cellpose-python",
+            decode_json=True
+        )
+        mask = ret['mask'][0]
+        print("Model test passed: ", "Cellpose using cellpose-python")
```
Comment (oeway) [inline scripts/test-triton-models.py:81]: Can we add some assert statements here to check the mask shape etc.? There should be a `success` key, and if it's false, there should be an `error` key to get the stack trace.

**Comment by oeway** [comment]
> Hey @oeway - I think this looks mostly reasonable now (though let me know if you want me to remove imjoy.io from the server list).

Looks good! I just added a comment. and we can keep ai.imjoy.io there.

File: scripts/test-triton-models.py (hunk 1 above)
Comment (jmetz) [reply scripts/test-triton-models.py:81]: Shouldn't the mask access fail if the process failed?

Comment (jmetz) [reply scripts/test-triton-models.py:81]: But yeah, can check for a success key and get it's value too

Comment (jmetz) [reply scripts/test-triton-models.py:81]: There is no success field... the return value is: 
```python
{
'mask': array([[0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0],
       ...,
       [0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0],
       [0, 0, 0, ..., 0, 0, 0]], dtype=int32), 
 '__info__': {
     'model_name': 'stardist', 
     'model_version': '1', 
     'outputs': [
         {'name': 'mask', 
           'datatype': 'INT32', 
           'shape': [349, 467], 
           'parameters': {'binary_data_size': 651932}
          }
       ]
    }
}
```

Comment (jmetz) [reply scripts/test-triton-models.py:81]: So checking that the mask exists and contains a 2d array is probably about as good a check as we can do. Presumably it should have the same shape as the input?

NB: this is `stardist` not the `bioengine-model-runner`

Comment (jmetz) [reply scripts/test-triton-models.py:81]: Just added shape tests... `stardist` was fine, but `cellpose-python` returns a stupid rolled-dimension array :roll_eyes:  : 

```
 (1, 349, 467) != (349, 467, 3)
 ```

Comment (jmetz) [reply scripts/test-triton-models.py:81]: Actually, ignore that. Upon closer paying attention, I get it :rofl: . The input is RGB, output is binary mask (and for some reason has to have the singleton dimension added).

**Pull request merged**